ui/components/issuesidebar/issuesidebar.go

```diff
@@ -243,6 +248,18 @@ func (m *Model) GetIsCommenting() bool {
 	return m.isCommenting
 }
 
+func (m *Model) shouldCancelComment() bool {
+	if !m.ShowConfirmCancel {
+		m.inputBox.SetPrompt("Really cancel comment? (Press Ctrl+c/esc again to cancel)")
```

Comment:
```suggestion
		m.inputBox.SetPrompt(lipgloss.NewStyle().Foreground(m.ctx.Theme.ErrorText).Render("Discard comment? (y/N)"))
```
I prefer having the user explicitly typing `y` to discard the comment.
So it flips the logic.